.eslintrc.js

```diff
@@ -91,7 +91,7 @@ module.exports = {
       "new-parens": "error",
       "newline-after-var": "error",
       "newline-before-return": "error",
-      "newline-per-chained-call": "error",
+      "newline-per-chained-call": "warn",
```

Comment:
I think we should keep our lint rules consistent and rather we should add an eslint ignore comment in the code if we want to skip the rule for this component. If we are in the habit of changing lint config files we will lose track of what is the standard.

```suggestion
      "newline-per-chained-call": "error",
```